sound/soc/sdw_utils/soc_sdw_utils.c

```diff
@@ -254,6 +254,25 @@ struct asoc_sdw_codec_info codec_info_list[] = {
 		},
 		.dai_num = 1,
 	},
+	{
+		.part_id = 0x1320,
+		.dais = {
+			{
+				.direction = {true, true},
+				.dai_name = "rt1320-aif1",
+				.dai_type = SOC_SDW_DAI_TYPE_AMP,
+				.dailink = {SOC_SDW_AMP_OUT_DAI_ID, SOC_SDW_AMP_IN_DAI_ID},
```

Comment:
@shumingfan @bardliao does the RT1320 really provide a feedback stream? I thought it was a companion amp, and today the existing RT722 does not provide such a feedback?

Reply:
It'd be best to be consistent, so that both main and companion amplifiers provide a feedback stream or none do.

Reply:
> It'd be best to be consistent, so that both main and companion amplifiers provide a feedback stream or none do.

Feedback link removed.